saleor/dashboard/product/forms.py

```diff
@@ -264,7 +264,7 @@ class Meta:
         labels = {
             'image': pgettext_lazy('Product image', 'Image'),
             'alt': pgettext_lazy(
-                'Short description', 'Short description')}
+                'Product image description', 'Short description')}
```

Comment:
Is this translation (like "krótki opis") likely to be specific to product images?